qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -257,6 +252,23 @@ def _draw_widgets(self):
                 self.prog.enabled = True
                 if tab:
                     self.prog.on_tab_changed(tab)
+            elif segment.startswith('text'):
+                cur_widget = textbase.TextBase()
+                self.text_widgets.append(cur_widget)
+                cur_widget.setText(segment.partition(':')[2])
+                self._hbox.addWidget(cur_widget)
+                cur_widget.show()
```

Comment:
While it should've here before, please add a:

```suggestion
                cur_widget.show()
            else:
                raise utils.Unreachable(segment)
```

so that we don't silently fail if somehow an unexpected value ends up here.